### ISSUE TITLE
Project Inventory: Food Oasis

### ISSUE THREAD

**Issue by Olivia-Chiong** [hackforla/product-management#130]
### Overview
We need to make an inventory for this project so we understand what assistance this project needs and what examples of effective processes or practices this team might have.

### Action Items
- [ ] Check the project for the items on the check list below
- [ ] Mark off the items when you find it and include the links where available
- [ ] Provide comments on what you find in the issue and also highlight when they have a great example so we can use it for the organization

**Product**
- [ ] [Readme]()
- [ ] [Wiki]()
- [ ] [Google Drive]()
- [ ] [GitHub Repository]()
   - [ ] [Project Board]()
   - [ ] [Issue Templates]() 
- [ ] [Slack Channel]()
- [ ] 1Password
- [ ] Project Email
   - [ ] projectname@hackforla.org
   - [ ] projectname@gmail.com (You will need this for setting up some applications like MIro that use rate limiting. Do not send emails using this address) 
- [ ] [Team Roster]()
- [ ] [Product One Sheet]()
- [ ] [Recruitment Form]()
- [ ] [OKRs and WBS]()
- [ ] [Roadmap]()
- [ ] [Storypoints]()
- [ ] [Meeting Minutes Template]()

**Marketing**
   - [ ] [Marketing Objectives]()
      - [ ] Create new issue using this template: [Define Marketing Objectives for [INSERT PROJECT NAME]](https://github.com/hackforla/marketing/issues/new?assignees=&labels=Level%3A+Learning%2C+Skill%3A+Brand+Strategy%2C+Skill%3A+Digital+Marketing%2C+Skill%3A+Media+Strategy%2C+documentation&template=define-marketing-objectives-for--insert-project-name-.md&title=Define+Marketing+Objectives+for+%5BINSERT+PROJECT+NAME%5D+project)
      - [ ] Copy the link for the issue into the parentheses above
   - [ ] [Collect Marketing Assets]()
      - [ ] Create new issue using this template: [Collect Marketing Assets for [INSERT PROJECT NAME]](https://github.com/hackforla/marketing/issues/new?assignees=&labels=feature%3A+marketing+inventory+checklist&template=collect-marketing-assets-for--insert-project-name-.md&title=Collect+Marketing+Assets+for+%5BINSERT+PROJECT+NAME%5D)
      - [ ] Copy the link for the issue into the parentheses above
   
**UI/UX**
- [ ] [Figma]()
- [ ] [Miro]()
- [ ] [Presentation Deck]()
- [ ] [Competitive Analysis]()
- [ ] [User Persona/Journeys]()
- [ ] [Communication Style Guide (including documents, spreadsheets and forms)]()
- [ ] [User Research Guide/Template]()
- [ ] [Wireframes Template]()
- [ ] [Website analytics software setup]()

### Resources/Instructions
REPLACE THIS TEXT -If there is a website which has documentation that helps with this issue provide the link(s) here.

**Comment by Olivia-Chiong** [comment]
@henlatourrette please work with @ktjnyc to work on getting the project inventory done for Food Oasis

**Comment by Olivia-Chiong** [comment]
@henlatourrette we are going to unassign you from this and make a new issue for you to do this for 100 Automations.

**Comment by henlatourrette** [comment]
Hi! OK, I will do it for 100 automations

**Comment by ktjnyc** [comment]
Linked Github, Github project board and slack channel.

**Comment by ktjnyc** [comment]
Progress: Updated links for Figma, Google Drive, Team Roster
Blocks: So many documents to sift through! 
Availability: variable
ETA: Can make some additional progress by next meeting

**Comment by ktjnyc** [comment]
Linked in user stories and updated project email

**Comment by ktjnyc** [comment]
Progress: Linked in issues for marketing assets and objectives, roadmap and one sheet.  Story point labels set up but many conflicting so we need to clean up and simplify.  
Blocks: So many documents to sift through!
Availability: variable
ETA: Can make some additional progress by next meeting

**Comment by ktjnyc** [comment]
Progress: Created an [issue ](https://github.com/hackforla/food-oasis/issues/921)on the Food Oasis to have the UX/IU team identify the assets.  We are working on the One Sheet and some of the others are in progress too so we don't have current versions yet.  
Blocks: So many documents to sift through!
Availability: variable
ETA: Can make some additional progress by next meeting

**Comment by wanderingspoon** [comment]
I have a question about the term "user stories" as applied and [linked](https://github.com/hackforla/food-oasis/wiki/User-Stories) above.

They currently read more like user roles than discrete units of functions/features for dev team's work. 

Or perhaps I'm looking at the wrong document? Could someone please clarify? Thanks!

Otherwise, I recommend that a separate issue is required for creating user stories based on issues-in-progress and prioritized backlog. I could help with those. Which can then go through the Fibonacci process.

**Comment by ktjnyc** [comment]
@wanderingspoon that should have been unlinked - John mentioned that he had just written that and posted it so we had something.   I've searched the rest of the Google drive and haven't found anything else.   We have an issue on the Food Oasis repo to [review and update the personas ](https://github.com/hackforla/food-oasis/issues/885).  We can create a separate one for user stories as well.

**Comment by gigicobos** [comment]
@Olivia-Chiong we'll discuss this tonight with the FOLA team to continue with this issue. We've added a link to our board as a reference.
